RELEASE_SUPPORT.md

```diff
@@ -11,20 +11,29 @@ By default, *eProsima Fast DDS* minor releases have a lifecycle of **6 months**.
 However, the life of certain branches can be extended depending on costumer and user needs.
 Within the support period of any minor version, there may be several patch releases that either add new functionalities in an ABI compatible way or fix possible issues.
 Before the minor release end of life (EOL) date, a final patch release including the latest fixes will be made available if there are any changes not included in any previous patch release of the same minor version.
+Long Term Supported (LTS) versions may have an extended supported period for only critical issues and security fixes, where no other ABI compatible neither bug fixes will be backported.
+This period applies since the end of standard support date to the end of life (EOL) date.
 
-The table below outlines the *eProsima Fast DDS* minor releases and their support cycles:
+The table below outlines the *eProsima Fast DDS* currently supported versions and their support cycles:
+
+|Version branch|Release Date|End of Standard Support Date|EOL Date|
```

Comment:
I propose two more columns to be added to these tables (on the left):

1. Version (like `2.14`, `2.13`, and so on)
2. Latest release (like `v2.14.2`, linked with the corresponding release notes)

Reply:
I've updated the release checklist to update it when new version released